### ISSUE TITLE
cpp-llama-python not found. 

### ISSUE THREAD

**Issue by NitkarshChourasia** [PromtEngineer/localGPT#782]
Whenever I try to ingest my documents. It shows me this error, *WHY*?

It also gives me the solution to `pip install cpp-llama-python` 

Can someone tell me what this is used for in the current project?

**Comment by PBJI** [comment]
Maybe you are wrong. @NitkarshChourasia, I spotted you thief.

**Comment by anabellechan** [comment]
Yes, llama-cpp-python is required for this program to work and you download it using pip install like you mentioned. It is a Python binding for llama-cpp and supports a number of hardware acceleration backends to speed up inference. cpp = C++ which is for efficient execution.
